src/include/rbd/librbd.h

```diff
@@ -1180,6 +1180,8 @@ CEPH_RBD_API int rbd_aio_flush(rbd_image_t image, rbd_completion_t c);
  */
 CEPH_RBD_API int rbd_invalidate_cache(rbd_image_t image);
 
+CEPH_RBD_API int rbd_discard_image_cache(rbd_image_t image);
```

Comment:
Actually -- can we just re-use the existing `rbd_invalidate_cache` API? I don't care for the overloaded usage of `discard` -- and having `rbd_invalidate_image_cache` seems superfluous.

Reply:
DONE